cmd/neofs-adm/internal/modules/morph/initialize_transfer.go

```diff
@@ -144,10 +141,12 @@ func (c *initializeContext) transferGASToProxy() error {
 		return err
 	}
 
-	tx, err := createNEP17MultiTransferTx(c.Client, c.CommitteeAcc, []rpcclient.TransferTarget{{
-		Token:   gas.Hash,
-		Address: proxyCs.Hash,
-		Amount:  initialProxyGASAmount,
+	from := c.CommitteeAcc.Contract.ScriptHash()
+
+	tx, err := createNEP17MultiTransferTx(c.Client, c.CommitteeAcc, gas.Hash, []nep17.TransferParameters{{
```

Comment:
Maybe then `createNEP17MultiTransferTx` can be removed completely? And you transfer it from the committee account anyway, so you can use committee actor directly from `c.CommitteeAct` without creating simple actor. Is there any pitfalls?

Reply:
> Maybe then createNEP17MultiTransferTx can be removed completely?

can be done, ok

> Is there any pitfalls?

do not think there are any

Reply:
If we've agreed to implement Builder-related changes in a separate PR, then let's fix this issue, and otherwise LGTM.

Reply:
> Maybe then createNEP17MultiTransferTx can be removed completely?

fixed